exporter/loadbalancingexporter/config.go

```diff
@@ -0,0 +1,43 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package loadbalancingexporter
+
+import (
+	"go.opentelemetry.io/collector/config/configmodels"
+	"go.opentelemetry.io/collector/exporter/otlpexporter"
+)
+
+// Config defines configuration for the exporter.
+type Config struct {
+	configmodels.ExporterSettings `mapstructure:",squash"`
+	OTLP                          otlpexporter.Config `mapstructure:"otlp"`
```

Comment:
Do you want to make this configurable in the future?

Reply:
In which sense? I think it's safe to assume that the first versions will support only OTLP, but I wanted to leave it on its own node, for both organization purposes and future-proofing.

Reply:
correct, was just thinking if we need another layer "rawexporter" which will implement a oneof idea.

Reply:
Similar to what we have in the jaeger receiver with "protocols" level.

Reply:
We'll ever only have one active exporter "type" per load balancer, even if we add support for different exporter types in the future. The `protocols` case for Jaeger is because one or more may be active at the same time for the instance.

Reply:
Correct, but I think it will be clear for user to have this oneof under a sub-level, we can do this later if you don't want to do it now. The main reason is that it isolates the section that deals with the protocol configuration:

```
protocol:
  otlp:
    ... 

OR

protocol:
  jaeger:
    ...
```

Reply:
Not sure I agree this is a great design, as it creates one more level that isn't really needed. In any case, I implemented this so that we can move forward.